CONTRIBUTING.md

```diff
@@ -114,6 +140,41 @@ Examples of PR titles:
  - test: Add infinity test for `MoveEffect.to`
  - refactor: Optimize the structure of the game loop
 
+
+# Maintainers
+
+These instructions are for the maintainers of Flame.
+
+
+## Merging a pull request
+
+When merging a pull request, make sure that the title of the merge commit has the correct
+conventional commit tag and a descriptive title. This is extra important since sometimes the title
+of the PR doesn't reflect what GitHub defaults to for the merge commit title (if the title has been
+changed during the life time of the PR for example).
+
+All the default text should be removed from the commit message and the PR description and the
+instructions from the "Migration instruction" (if the PR is breaking) should be copied into the
+commit message.
+
+
+## Creating a release
+
+There are a few things to think about when doing a release:
+ 
+ - Search through the codebase for `@Deprecated` methods/fields and remove the ones that are marked
+   for removal in the version that you are intending to release.
+ - Create a PR containing the changes for removing the deprecated entities.
+ - Run `melos version -V <package1>:<version> -V <package2>:<version>` for Melos to generate a
+   `CHANGELOG.md` file.
```

Comment:
```suggestion
   `CHANGELOG.md` file for each package you provide.
```

To make it more clear in general? Cause that was what originally confused it, that it was only for a single package

Reply:
It is also generating a general changelog, maybe "file" should just be changes to "files"?

Reply:
Yea that sounds much better if it does a general one aswell!